Automattic/crowdsignal-ui
# Refactor block related packages

This patch introduces some changes in the organization for our block packages:

- The original `@crowdsignal/block-editor` package has been removed as it's now left unused following #45
- `@crowdsignal/blocks` have been renamed to `@crowdsignal/block-editor`, it will contain the editor blocks and any strictly associated things
- A new package has been created to replace `@crowdsignal/blocks` which is meant to host the implementations for the public-facing blocks as well as any common elements shared by the editor version as well.
- `renderBlocks()` has been moved from the original `@crowdsignal/blocks` into the new `@crowdsignal/blocks` as it's not meant to be used with the editor.

# Testing

- Run `yarn install`
- Run `yarn workspace @crowdsignal/dashboard start` and verify the editor and our existing blocks still work as expected
- Run `yarn storybook` and verify the poll widget still displays
